src/benchmarks/micro/corefx/System.Text.Json/Serializer/ReadJson.GenericCollection.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using System.Collections.Generic;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public class WriteGenericCollection
```

Comment:
The other PR (https://github.com/dotnet/performance/pull/530) has `ReadDictionary` even though it too is a generic collection. Why is this not `WriteSet` instead rather than `WriteGenericCollection` (which is, too generic of a name), to be consisten?

Reply:
I plan to add other generic collections (e.g. `IEnumerable<T>,` `Stack<T>`, etc.) to these classes, so I'm renaming them `Read/WriteGenericCollections`.
I have dictionary in it's own classes because it makes sense to me to group the several dict types across the dictionary namespaces together.